packages/ncform/src/components/vue-ncform/form-item.vue

```diff
@@ -277,7 +277,7 @@ export default {
 
         // 对比控件改变前后的值，判断是否需要对其进行校验。
         if (changed) {
-          if (!ncformUtils.isNormalObjSchema(newVal) && !ncformUtils.isNormalArrSchema(newVal)) { // 叶子结点
```

Comment:
Why remove `!ncformUtils.isNormalArrSchema(newVal)` ? object type with `properties` field and array type with `items` field, these are all not the leaf node

Reply:
I think not only the leaf node needs to trigger the change event, but also the array itself, like add an item field.